datasets/gg/disqualified_directors/gg_disqualified_directors.yaml

```diff
@@ -1,37 +1,38 @@
-title: Guernsey Disqualified Directors
+title: Guernsey Financial Services Commission - Prohibitions and Disqualified Directors
 entry_point: crawler.py
-prefix: gg-dir
+prefix: gg-gfsc
 coverage:
   frequency: daily
   start: "2024-07-26"
 load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
 summary: >
-  Company directors who have been disqualified by the Court
+  List of individuals disqualified as company directors or prohibited from specific functions 
+  by the Guernsey Financial Services Commission. 
 description: |
-  The Royal Court of Guernsey has the power to disqualify individuals from
-  being appointed as a director or other officer of a company, or participating
-  in the formation, management or promotion of a company
-  (Companies (Guernsey) Law 2008).
-
-  This dataset lists persons disqualified in this way as published by the Guernsey Registry.
+  This dataset includes two separate lists: one of individuals disqualified from being company 
+  directors under the Companies (Guernsey) Law 2008, and another of prohibition orders issued 
+  by the Guernsey Financial Services Commission, detailing individuals prohibited from specific 
+  functions since 1 November 2021.
```

Comment:
```suggestion
  This dataset includes two separate lists: one of individuals disqualified from being company 
  directors under the Companies (Guernsey) Law 2008, and another of prohibition orders issued 
  by the Guernsey Financial Services Commission, detailing individuals prohibited from specific 
  functions since 1 November 2021.
  
  Disqualified directors are sourced from the [Guernsey Registry](https://www.guernseyregistry.com/article/4036/Disqualified-Directors).
```